README.md

```diff
@@ -170,6 +178,21 @@ Do not hesitate to contact
 need help with the integration of your PDE solver.
 
 
+External Matrix Equation Solvers
+--------------------------------
+
+pyMOR also integrates matrix equation solvers for system-theoretic methods.
+The following solver libraries are included:
+
+* [M.E.S.S.](https://www.mpi-magdeburg.mpg.de/projects/mess)
+
+    The Matrix Equation Sparse Solver library is intended for solving large sparse matrix equations.
+
+* [Slycot](https://github.com/python-control/Slycot)
+
+    Python wrapper for the Subroutine Library in Systems and Control Theory (SLICOT). 
```

Comment:
Mention `pymor.bindings.slycot`?
Also, Slycot is used in H_inf-norm computation (`pymor.discretizations.LTISystem.hinf_norm`).